src/Engine/GameObject/GameObject.cpp

```diff
@@ -23,7 +23,7 @@ GameObject::GameObject(double x_pos, double y_pos, double x_vel, double y_vel, i
 }
 
 GameObject::~GameObject() {
-    LOG(INFO) << "GameObject with id: " << id_ << " died";
+    DLOG(INFO) << "GameObject with id: " << id_ << " died";
```

Comment:
what's this?

Reply:
I remember you suggested before that you wanted some stuff in DEBUG. well `LOG` doesn't have debug, you have to call `DLOG`